src/main/java/org/dependencytrack/persistence/jdbi/FindingDao.java

```diff
@@ -0,0 +1,560 @@
+               , "VULNERABILITY"."SEVERITY"
+               , "VULNERABILITY"."CVSSV2BASESCORE"
+               , "VULNERABILITY"."CVSSV3BASESCORE"
+               , "FINDINGATTRIBUTION"."ANALYZERIDENTITY"
+               , "VULNERABILITY"."PUBLISHED"
+               , "VULNERABILITY"."CWES"
+            <#if apiOrderByClause??>
+              ${apiOrderByClause}
+            </#if>
+            """)
+    @AllowApiOrdering(by = {
+            @AllowApiOrdering.Column(name = "vulnerability.vulnId", queryName = "\"VULNERABILITY\".\"VULNID\""),
+            @AllowApiOrdering.Column(name = "vulnerability.title", queryName = "\"VULNERABILITY\".\"TITLE\""),
+            @AllowApiOrdering.Column(name = "vulnerability.severity", queryName = """ 
+                     CASE WHEN "VULNERABILITY"."SEVERITY" = 'UNASSIGNED' 
+                          THEN 0 
+                          WHEN "VULNERABILITY"."SEVERITY" = 'LOW' 
+                          THEN 3 
+                          WHEN "VULNERABILITY"."SEVERITY" = 'MEDIUM' 
+                          THEN 6 
+                          WHEN "VULNERABILITY"."SEVERITY" = 'HIGH' 
+                          THEN 8 
+                          WHEN "VULNERABILITY"."SEVERITY" = 'CRITICAL' 
+                          THEN 10 
+                          ELSE CASE WHEN "VULNERABILITY"."CVSSV3BASESCORE" IS NOT NULL 
+                                    THEN "VULNERABILITY"."CVSSV3BASESCORE" 
+                                    ELSE "VULNERABILITY"."CVSSV2BASESCORE" 
+                               END 
+                     END 
+                     """),
+            @AllowApiOrdering.Column(name = "vulnerability.cvssV3BaseScore", queryName = "\"VULNERABILITY\".\"CVSSV3BASESCORE\""),
+            @AllowApiOrdering.Column(name = "vulnerability.cvssV2BaseScore", queryName = "\"VULNERABILITY\".\"CVSSV2BASESCORE\""),
+            @AllowApiOrdering.Column(name = "vulnerability.published", queryName = "\"VULNERABILITY\".\"PUBLISHED\""),
+            @AllowApiOrdering.Column(name = "attribution.analyzerIdentity", queryName = "\"FINDINGATTRIBUTION\".\"ANALYZERIDENTITY\""),
+            @AllowApiOrdering.Column(name = "vulnerability.affectedProjectCount", queryName = "affectedProjectCount")
+    })
+    @AllowUnusedBindings
+    @RegisterConstructorMapper(GroupedFindingRow.class)
+    List<GroupedFindingRow> getGroupedFindings(@Define String queryFilter, @Define boolean activeFilter);
+
+    /**
+     * Returns a List of all Finding objects filtered by ACL and other optional filters. The resulting list is grouped by vulnerability.
+     *
+     * @param filters       determines the filters to apply on the list of Finding objects
+     * @param showInactive  determines if inactive projects should be included or not
+     * @return a List of Finding objects
+     */
+    default PaginatedResult getGroupedFindings(final AlpineRequest alpineRequest, final Map<String, String> filters, final boolean showInactive) {
```

Comment:
Sorting by affected projects is broken:

![image](https://github.com/user-attachments/assets/cd25c38f-785c-4a09-be3e-c3756fce5dfb)


```
dt-apiserver  | 2025-03-19 11:42:23,977 ERROR [GlobalExceptionHandler] Uncaught internal server error [requestId=2281333f-ef33-4b39-801d-9a958ff90e72]
dt-apiserver  | org.jdbi.v3.core.statement.UnableToExecuteStatementException: org.postgresql.util.PSQLException: ERROR: column "affectedprojectcount" does not exist
```